eth/handler.go

```diff
@@ -284,7 +284,12 @@ func newHandler(config *handlerConfig) (*handler, error) {
 			}
 			return 0, nil
 		}
-		return h.chain.InsertChain(blocks)
+		n, err := h.chain.InsertChain(blocks)
+		// PTAL(meowsbits) This chunk was removed upstream. (No acceptTxs.Store...)
```

Comment:
Nice that you've added this comment here so as we don't miss it on subsequent merges.